data/json/mutations.json

```diff
@@ -4167,6 +4168,23 @@
         "thirst"     :    true
     },{
         "type" : "mutation",
+        "id" : "SLIMESPAWNER2",
+        "name" : "Mitosis",
+        "points" : 10,
+        "visibility" : 0,
+        "ugliness" : 0,
+        "description" : "Your body has changed further to allow you to create imperfect replicas of yourself. They are obedient and more useful than previous iterations, but also more resource intensive.",
+        "valid" : false,
+        "purifiable" : false,
+        "prereqs" : ["SLIMESPAWNER"],
+        "threshreq" : ["THRESH_SLIME"],
+        "category" : ["MUTCAT_SLIME"],
+        "active"     :    true,
+        "cost"       :    100,
```

Comment:
That cost is pretty low if we assume the created critter is roughly human-sized.
Thirst above 260 prevents using mutations and above 1200 causes death.